### PR TITLE
Hari/refactor/html

### PR DESCRIPTION
**Pull request by balaharikrishna**
Removed inline CSS and Seperated common CSS from media queries and reduced the code duplication.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -71,7 +72,7 @@
     </div>
 
     <div class="grid-item2">
-      <p style="color: #67a435;margin-top: 15px;" class="griditemmobp1">
+      <p  class="gls griditemmobp1">
```
Comment (akashdoppalapudi) [inline index.html]: have meaningful names

File: styles.css [hunk 2]
```diff
@@ -5,299 +5,152 @@ body {
...
+.buttonhomenav {
+  background-color: #7ba435;
+  border: none;
+  color: white;
+  padding: 5px 32px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 16px;
+  margin: 0px 0px;
```
Comment (akashdoppalapudi) [inline styles.css]: margin: 16% 0 0 0;

use this

File: styles.css [hunk 3]
```diff
@@ -5,299 +5,152 @@ body {
...
+  background-color: #7ba435;
+  border: none;
+  color: white;
+  padding: 5px 20px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 16px;
+  margin: 0px 0px;
+  cursor: pointer;
```
Comment (akashdoppalapudi) [inline styles.css:77]: margin: 8% 0 0 0;

File: styles.css [hunk 4]
```diff
@@ -328,78 +170,29 @@ body {
...
-    margin-bottom: -5px;
-  }
-
   .arrowbuttonnav {
-    background-color: #7ba435;
-    border: none;
-    color: white;
     padding: 5px;
     padding-right: 5%;
     padding-left: 5%;
```
Comment (akashdoppalapudi) [inline styles.css]: padding: 0 5%;

File: styles.css [hunk 5]
```diff
@@ -328,78 +170,29 @@ body {
...
-    text-align: center;
-    text-decoration: none;
-    display: inline-block;
-    font-size: 16px;
     margin: 4px 2px;
-    cursor: pointer;
-    border-radius: 10px;
     margin: 0;
```
Comment (akashdoppalapudi) [inline styles.css:137]: margin: 0; will rewrite the styles specified above

File: styles.css [hunk 6]
```diff
@@ -411,445 +204,238 @@ body {
...
+.arrowbutton {
+  background-color: #7ba435;
+  border: none;
+  color: white;
+  padding: 5px 20px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 16px;
+  margin: 0px 0px;
```
Comment (akashdoppalapudi) [inline styles.css]: margin: 0;

File: styles.css [hunk 7]
```diff
@@ -928,564 +481,235 @@ body {
...
+  line-height: 1.5em;
+  margin-top: auto;
+  margin-bottom: auto;
+  font-weight: 600;
+}
+.UpMatchcont {
+  text-align: center;
+  line-height: 1em;
+  margin-top: auto;
+  margin-bottom: auto;
```
Comment (akashdoppalapudi) [inline styles.css]: margin: auto 0;

File: styles.css [hunk 8]
```diff
@@ -928,564 +481,235 @@ body {
...
+.cardbottom{
+  margin-bottom: 30px;
+}
+.cardtextdate{
+  margin-top: 40px;
+}
+.grid-container4-0 {
+  display: grid;
+  padding-left: 5%;
+  padding-right: 5%;
```
Comment (akashdoppalapudi) [inline styles.css]: padding: 0 5%;

**Review by akashdoppalapudi — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (balaharikrishna) [reply index.html]: updated the Class Name "gls" to "playbuttongamesection".

File: styles.css (hunk 2 above)
Comment (balaharikrishna) [reply styles.css]: Updated the CSS.

File: styles.css (hunk 3 above)
Comment (balaharikrishna) [reply styles.css:77]: Updated the CSS.

File: styles.css (hunk 4 above)
Comment (balaharikrishna) [reply styles.css]: Updated the CSS.

File: styles.css (hunk 5 above)
Comment (balaharikrishna) [reply styles.css:137]: Updated the CSS.

File: styles.css (hunk 6 above)
Comment (balaharikrishna) [reply styles.css]: Updated the CSS as said.

File: styles.css (hunk 7 above)
Comment (balaharikrishna) [reply styles.css]: Updated the CSS.

File: styles.css (hunk 8 above)
Comment (balaharikrishna) [reply styles.css]: Updated the CSS.

File: index.html (hunk 1 above)
Comment (balaharikrishna) [reply index.html]: updated the Class Name "gls" to "playbuttongamesection".

**Review by akashdoppalapudi — APPROVED** [review]
lgtm

**Pull request merged**